rpcserver.go

```diff
@@ -1099,9 +1101,17 @@ func (r *rpcServer) sendCoinsOnChain(paymentMap map[string]int64,
 
 	// If that checks out, we're fairly confident that creating sending to
 	// these outputs will keep the wallet balance above the reserve.
-	tx, err := r.server.cc.Wallet.SendOutputs(
-		outputs, feeRate, minConfs, label, strategy,
-	)
+	var tx *wire.MsgTx
+	if len(selectedUtxos) == 0 {
```

Comment:
This `if/else` is another good reason to just add a variadic/functional option to the `SendOutputs` method.